debugging/segfault.ipynb

```diff
@@ -0,0 +1,218 @@
+{
```

Comment:
I think you can delete this file - cytopipe will be a python package.

If meant to include this file, please prefix this folder with `analysis/` (so `analysis/debugging/segfault.ipynb`). Adding the analysis prefix makes the separation from python package more explicit

Reply:
debugging folder will be deleted. It was used for me to conduct live debugging when ever I found bugs. 

Debugging has no “usage” purpose. It was used for my purposes to figure out the segault issues. This has been removed on commit : 67828fb3af04336c529db4eebfd2c0f8f3958e04